gst-libs/mfx/gstmfxwindow.c

```diff
@@ -118,19 +118,19 @@ gst_mfx_window_class_init (GstMfxWindowClass * klass)
 
 GstMfxWindow *
 gst_mfx_window_new_internal (GstMfxWindow *window, GstMfxContext * context,
-  GstMfxID id, guint width, guint height)
+  /*GstMfxID id,*/ guint width, guint height)
```

Comment:
I left this deliberately anticipating merging both Windows and Linux code bases, since X11 has the concept of foreign windows. I wasn't sure that time that Windows too had such a concept but I think it's better to leave it there to generalize the class better.